platforms/chibios/converters/promicro_to_stemcell/_pin_defs.h

```diff
@@ -0,0 +1,57 @@
+// Copyright 2022 Mega Mind (@megamind4089)
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#pragma once
+
+// Pindefs for v2.0.0
+// https://megamind4089.github.io/STeMCell/pinout/
+
+// Left side (front)
+#ifdef STEMCELL_UART_SWAP
```

Comment:
@megamind4089 - I think you have this #ifdef / #else backwards. I double checked against prior code, and it was the other way around. I noticed because on one of my boards, I'm using the RX pin (D2) as a row input, and it stopped working

Should be:
```
#ifdef STEMCELL_UART_SWAP
#    define D3 PAL_LINE(GPIOA, 3)
#    define D2 PAL_LINE(GPIOA, 2)
#else
#    define D3 PAL_LINE(GPIOA, 2)
#    define D2 PAL_LINE(GPIOA, 3)
#endif
```

Reply:
Thanks @sadekbaroudi 
I did this intentionally, since most keyboards use D2 as split pin.
But as you suggested, this will not align with the hw jumper settings and will add confusion.
Reverted back the changes.